dapp/src/sagas/communityEntities.js

```diff
@@ -269,23 +272,6 @@ export function * fetchFeaturedCommunityEntitiesCount ({ communityAddress }) {
   }
 }
 
-function * fetchUserMetadata ({ account }) {
```

Comment:
So we need this or node?

Reply:
No, It used to fetch the user profile from 3box.

Reply:
So can you delete the whole function? Of you prefer to keep the changes minimal to just comment the body?

Reply:
I've deleted the function.